cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -31,4 +51,26 @@ describe('ui-inventory: keyboard shortcut', () => {
     InventoryMainButton.waitModalLoading('Keyboard shortcuts');
     InventoryMainButton.verifyInventoryDropdownIsShown('false');
   });
+
+  it('C345298 Keyboard Shortcut. Test the functionality of the different shortcut keys', { tags: [testTypes.smoke] }, () => {
+    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+
+    InventoryMainButton.openInventoryMenu();
+    InventoryMainButton.openShortcuts();
+
+    InventoryMainButton.waitModalLoading('Keyboard shortcuts');
+    InventoryMainButton.pressHotKey(hotKeys.create);
+    InventoryMainButton.closeShortcuts();
+    InventoryMainButton.fillInstanceInfo();
+    // TO DO: Need to wait for the loading of saving the edited information.Reason: the robot runs quickly and the test drops.4 sec, becouseI checked all time-outs like 1,2,3
+    cy.wait(4000);
```

Comment:
if 3 sec is not enough, I suggest to wait 6 seconds instead of 4. And have you checked responses waiters(we don't have another way then static waiter)?

Reply:
Done